Cargo.toml

```diff
@@ -129,3 +129,4 @@ verbose_file_reads = "warn"
 
```

Comment:
Commit title should be prefixed with `test:`, otherwise this ends up in the release notes as a feature.

Reply:
Specifically for the commit that adds tests, of course.